packages/terra-toggle-section-header/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "terra-toggle-section-header",
+  "main": "lib/ToggleSectionHeader.js",
+  "version": "0.0.0",
+  "description": "Toggle section header component that transitions content in and out with a click.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-toggle-section-header",
+    "ToggleSectionHeader",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^3.1.0",
+    "terra-mixins": "^1.13.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^3.1.0",
+    "terra-mixins": "^1.13.0",
+    "terra-section-header": "^0.x",
```

Comment:
Has section-header not been released yet?

Reply:
I don't believe so, [referernce](https://github.com/cerner/terra-core/pull/1339/files/3227c5ad63735913493f2247d14bef7cb99ed59a#r174142652). Doesn't terra release at v. `1.0.0`?

Reply:
It has not been released yet.